src/applications/static-pages/facilities/vetCentersHours.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO fix spacing and add unit tests
+const VetCenterHours = props => {
+  const arrayOfWeekdays = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+
+  const formatTime = hoursString => {
```

Comment:
We should definitely write a unit test for this function. It probably should live in `src/applications/facility-locator/utils`.